python/llm/README.md

```diff
@@ -115,8 +115,8 @@ You may run the models using `transformers`-style API in `bigdl-llm`.
       outfile='/path/to/output/', outtype='int4', model_family="llama")
```

Comment:
BTW, the indentation of "- ##### Using native INT4 format" above is wrong

Reply:
> BTW, the indentation of "- ##### Using native INT4 format" above is wrong

oh, I modified the indentation now, And another typo `lm-convert` in readme is also fixed.